src/main/java/com/iotiq/application/service/CustomerService.java

```diff
@@ -66,14 +68,14 @@ public void update(CustomerUpdateRequest request) {
     }
 
     @Transactional
-    public Customer getCurrentCustomerOrCreate() {
+    public Customer getCurrentCustomer() {
         User currentUser = userService.getCurrentUser();
```

Comment:
Using the principal from the security context might be better than calling this function everywhere. @oguzIotiq

Reply:
https://github.com/Iotiq-Erste/kimono-application/issues/60